### PR TITLE
M sc1 61 19

### PR DESCRIPTION
**Pull request by AliaksandrTratsiak**
css

### CODE REVIEW

File: atretiak/04_css/письмо/письмо.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Пример сопроводительного письма</title>
+        <link type="text/css" rel="stylesheet" href="index.css">
```
Comment (pluhin) [inline atretiak/04_css/письмо/письмо.html]: I do not see link for font from google

**Review by pluhin — CHANGES_REQUESTED** [review]
Do not use Cyrillic  and whitespase in folder/file names. Could you please rename all file

File: atretiak/04_css/письмо/письмо.html (hunk 1 above)
Comment (AliaksandrTratsiak) [reply atretiak/04_css/письмо/письмо.html]: fix typo css

**Review by AliaksandrTratsiak — COMMENTED** [review]
fix typo css

**Review by pluhin — APPROVED** [review]
Good, approved

**Pull request merged**